docs/backups-storage.md

```diff
@@ -0,0 +1,276 @@
+
+            === "Linux"
+
+                ```
+                base64 --wrap=0 <filename>
+                ```
+
+            === "MacOS"
+                
+                ```
+                base64 <filename>
+                ```
+
+        * Create the Kubernetes Secret configuration file and specify your cluster name
+        and the base64-encoded contents of the files from previous steps. The following is the example of the
+        `cluster1-pgbackrest-secrets.yaml` Secret file:    
+
+           ```yaml
+           apiVersion: v1
+           kind: Secret
+           metadata:
+             name: cluster1-pgbackrest-secrets
+           type: Opaque
+           data:
+             gcs-key.json: <base64-encoded-json-file-contents>
+             gcs.conf: <base64-encoded-conf-file-contents>
+           ```       
+
+           !!! note       
+
+               This Secret can store credentials for several repositories presented as
+               separate data keys.    
+
+    4. Create the Secrets object from the Secret configuration file:    
+
+        ``` {.bash data-prompt="$" }
+        $ kubectl apply -f cluster1-pgbackrest-secrets.yaml
+        ```    
+
+    5. Update your `deploy/cr.yaml` configuration. Specify your GCS credentials
+        Secret in the `backups.pgbackrest.configuration` subsection, and put GCS
+        bucket name into the `bucket` option 
+        in the `backups.pgbackrest.repos` subsection. The repository name must be the same as the name you specified when you created the `gcs.conf` file. For example, GCS storage configuration
+        for the `repo3` repository would look as follows.    
+
+        ```yaml
+        ...
+        backups:
+          pgbackrest:
+            ...
+            configuration:
+              - secret:
+                  name: cluster1-pgbackrest-secrets
+            ...
+            repos:
+            - name: repo3
+              gcs:
+                bucket: "<YOUR_GCS_BUCKET_NAME>"
+        ```    
+
+    6. Create or update the cluster:    
+
+        ``` {.bash data-prompt="$" }
+        $ kubectl apply -f deploy/cr.yaml
+        ```
+
+=== "Azure Blob Storage (tech preview)"
+
+    To use [Microsoft Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/) for storing backups, you need the following:
+
+    * a proper Azure container name. 
+    * Azure Storage credentials. These are stored in an encoded form in the [Kubernetes Secret](https://kubernetes.io/docs/concepts/configuration/secret/).
+
+    **Configuration steps**
+
+    1. Encode the Azure Storage credentials and the pgBackRest repo name that you will use for backups with base64. In this example, we are using `repo1`.   
+
+        === "Linux"    
+
+            ``` {.bash data-prompt="$" }
+            $ cat <<EOF | base64 --wrap=0
+            [global]
+            repo1-azure-account=<AZURE_STORAGE_ACCOUNT_NAME>
+            repo1-azure-key=<AZURE_STORAGE_ACCOUNT_KEY>
```

Comment:
In cr azure it is repo3 by default
could we also use in key names repo3? 
My point people will use default settings (just uncomment rows in cr)

Reply:
Updated repo names according to cr.yaml